src/adapters/collateralized_simple_interest_loan_adapter.ts

```diff
@@ -1,25 +1,35 @@
 import * as Web3 from "web3";
 import * as singleLineString from "single-line-string";
 import * as omit from "lodash.omit";
+import * as moment from "moment";
 
-import { BigNumber } from "../../utils/bignumber";
+import { BigNumber } from "utils/bignumber";
+import { Web3Utils } from "utils/web3_utils";
 
-import { ContractsAPI } from "../apis";
-import { Assertions } from "../invariants";
-import { DebtOrder, DebtRegistryEntry, RepaymentSchedule } from "../types";
-import { Adapter } from "./adapter";
+import { ContractsAPI } from "src/apis";
+import { Assertions } from "src/invariants";
+import { DebtOrder, DebtRegistryEntry, RepaymentSchedule } from "src/types";
 
+import { Adapter } from "./adapter";
 import { TermsContractParameters } from "./terms_contract_parameters";
+
 import {
     SimpleInterestLoanTerms,
     SimpleInterestLoanOrder,
     SimpleInterestTermsContractParameters,
 } from "./simple_interest_loan_adapter";
 
+import { TermsContract } from "src/wrappers";
+
+import { NULL_ADDRESS } from "../../utils/constants";
+import { ACCOUNTS } from "../../__test__/accounts";
```

Comment:
These are test accounts -- they are not available to us in a production context.